scripts/rhel_validation/prepare_lis_vm.sh

```diff
@@ -37,6 +38,15 @@ function prepare_vm {
 	    printf "\n kernel-${kernel_version} installation failed \n"
     fi
     yum -y install kernel-devel-${kernel_version}
+
+    yum -y install yum-utils
+    yumdownloader -y kernel-${kernel_version} kernel-devel-${kernel_version}
+
+    wget https://aka.ms/downloadazcopy-v10-linux
```

Comment:
So next time, if we have newer version like v11, then should we change this script again?

Reply:
This AzCopy download link is managed by microsoft as official download link.
Even if they release v11, this link will continue to work.
Reference : https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10

Reply:
Otherwise, we can replace aka.ms link with actual download link - 
https://azcopyvnext.azureedge.net/release20190517/azcopy_linux_amd64_10.1.2.tar.gz

Reply:
@sireeshapotnuru please change the code handling this part against continuous changes.